.circleci/src/workflows/commit.yml

```diff
@@ -25,7 +25,8 @@ jobs:
       build_config_name: integreat-e2e
       context:
         - tuerantuer-google-play
-        - tuerantuer-android
+        - credentials-repo
+        - credentials-integreat
```

Comment:
Why did you split it? Which contains which credentials?

Reply:
credentials-repo can be reused accross different projects in the Integreat organisation. `credentials-integreat` contains integreat-specific vars.